src/components/page-layout/Layout.js

```diff
@@ -18,28 +18,32 @@ import styles from './styles/layout.module.css';
 
 const Layout = ({ children }) => {
   const data = useStaticQuery(graphql`
-        query SiteTitleQuery {
-            site {
-                siteMetadata {
-                    title
-                }
-            }
+    query SiteTitleQuery {
+      site {
+        siteMetadata {
+          title
         }
-    `);
+      }
+    }
+  `);
 
   return (
-        <>
-            <Header siteTitle={data.site.siteMetadata.title} />
-            <NavigationBar />
-            <div className={styles.body}>
-                <main>{children}</main>
-                <footer>
-                    © {new Date().getFullYear()}, Built with
-                    {' '}
-                    <a href="https://www.gatsbyjs.org">Gatsby</a>
-                </footer>
-            </div>
-        </>
+    <>
+      <Header siteTitle={data.site.siteMetadata.title} />
+      <NavigationBar />
+      <div className={styles.body}>
+        <main>{children}</main>
+        <footer>
+          <p>
+            ©2020, A <a href="https://www.viget.com/">Viget</a> Project
```

Comment:
Quick trick when setting copyright dates — grab the current year instead with `${new Date().getFullYear()}`. That way, you never need to update it.